pkg/pool-hooks/contracts/ECLPSurgeHook.sol

```diff
@@ -20,18 +21,18 @@ import { SurgeHookCommon } from "./SurgeHookCommon.sol";
  * @notice Hook that charges a fee on trades that push a pool into an imbalanced state beyond a given threshold.
  * @dev Uses the dynamic fee mechanism to apply a "surge" fee on trades that unbalance the pool beyond the threshold.
  */
-contract ECLPSurgeHook is SurgeHookCommon {
+contract ECLPSurgeHook is IECLPSurgeHook, SurgeHookCommon {
     using SignedFixedPoint for int256;
     using FixedPoint for uint256;
     using SafeCast for *;
 
-    /**
-     * @notice A new `ECLPSurgeHook` contract has been registered successfully.
-     * @dev If the registration fails the call will revert, so there will be no event.
-     * @param pool The pool on which the hook was registered
-     * @param factory The factory that registered the pool
-     */
-    event ECLPSurgeHookRegistered(address indexed pool, address indexed factory);
+    // Cannot use FixedPoint.ONE because the constant is a uint128.
+    uint128 internal constant _DEFAULT_IMBALANCE_SLOPE = 1e18;
+    uint128 public constant MAX_IMBALANCE_SLOPE = 10000e16;
+    uint128 public constant MIN_IMBALANCE_SLOPE = 1e16;
```

Comment:
```suggestion
    uint128 public constant MAX_IMBALANCE_SLOPE = 100e18;
    uint128 public constant MIN_IMBALANCE_SLOPE = 0.01e18;
```
Since these aren't percentages (what we usually use e16 for), maybe this is clearer?